tools/data/kinetics/README.md

```diff
@@ -63,7 +63,7 @@ For better decoding speed, you can resize the original videos into smaller sized
 python ../resize_videos.py ../../../data/${DATASET}/videos_train/ ../../../data/${DATASET}/videos_train_256p_dense_cache --dense --level 2
 ```
 
-You can also download from [Academic Torrents](https://academictorrents.com/)([kinetics400](https://academictorrents.com/details/184d11318372f70018cf9a72ef867e2fb9ce1d26) & [kinetics700](https://academictorrents.com/details/49f203189fb69ae96fb40a6d0e129949e1dfec98) are avaiable) and [cvdfoundation/kinetics-dataset](https://github.com/cvdfoundation/kinetics-dataset)(Host by Common Visual Data Foundation and Kinetics400/Kinetics600/Kinetics-700-2020 are available)
+You can also download the video datasets with short edge 256 (256p) from [Academic Torrents](https://academictorrents.com/)([kinetics400](https://academictorrents.com/details/184d11318372f70018cf9a72ef867e2fb9ce1d26) & [kinetics700](https://academictorrents.com/details/49f203189fb69ae96fb40a6d0e129949e1dfec98) are avaiable) and [cvdfoundation/kinetics-dataset](https://github.com/cvdfoundation/kinetics-dataset)(Host by Common Visual Data Foundation and Kinetics400/Kinetics600/Kinetics-700-2020 are available)
```

Comment:
Do Academic Torrents and cvdfoundation both host 256p?

Reply:
Checked a few videos from cvdfoundation and they are not resize to shortside 256

Reply:
maybe shortside 360

Reply:
hey, I have double-checked the videos in cvdfoundation and found some videos with short edge 720, maybe they are not resized to a fixed length.